composer_local_dev/environment.py

```diff
@@ -85,6 +87,11 @@ def get_default_environment_variables(
         "AIRFLOW__CORE__PLUGINS_FOLDER": "/home/airflow/gcs/plugins",
         "AIRFLOW__WEBSERVER__RELOAD_ON_PLUGIN_CHANGE": "True",
         "COMPOSER_PYTHON_VERSION": "3",
+        # By default, the container runs as the user `airflow` with UID 999. Set
+        # this env variable to "True" to make it run as the current host user.
+        "COMPOSER_CONTAINER_RUN_AS_HOST_USER": "False",
+        "COMPOSER_HOST_USER_NAME": f"{getpass.getuser()}",
+        "COMPOSER_HOST_USER_ID": f"{os.getuid()}",
```

Comment:
Won't this break on Windows?

Reply:
Good catch. I just confirmed `os.getuid()` is only available on Unix/Linux. https://docs.python.org/3/library/os.html#os.getuid

Any idea how to handle Windows? Should I adde a check for Windows and disallow the feature?

Reply:
Yeah, that would be great :)